src/ngraph/validation_util.hpp

```diff
@@ -102,4 +102,12 @@ namespace ngraph
                                    const AxisSet& new_axis_mask,
                                    const AxisSet& shrink_axis_mask,
                                    const AxisSet& ellipsis_mask);
+
+    int64_t normalize_axis(const Node* node, std::int64_t axis, std::int64_t tensor_rank);
+
+    int64_t normalize_axis(const Node* node,
+                           std::int64_t axis,
+                           std::int64_t tensor_rank,
+                           std::int64_t axis_range_min,
+                           std::int64_t axis_range_max);
```

Comment:
Since you have added these functions please remove the old ones(`validate_axis`) from `onnx_importer`.

Reply:
Done. `onnx_impoter` now uses ngraph version of `normalize_axis` .